app/controllers/counselor/counselors_controller.rb

```diff
@@ -10,7 +10,25 @@ def edit
   end
 
   def update
-    puts "GOT INTO UPDATE"
+    @counselor = Counselor.find_by(user_id: current_user.id)
+    @user = User.find(@counselor.user_id)
```

Comment:
Use `find_by` to avoid an error causing a crash.
I noticed there aren't any error checks either. But might be okay.